src/tools/genpolicy/rules.rego

```diff
@@ -55,12 +55,125 @@ default AllowRequestsFailingPolicy := false
 S_NAME_KEY = "io.kubernetes.cri.sandbox-name"
 S_NAMESPACE_KEY = "io.kubernetes.cri.sandbox-namespace"
 BUNDLE_ID = "[a-z0-9]{64}"
+# from https://github.com/kubernetes/kubernetes/blob/8294abc599696e0d1b5aa734afa7ae1e4f5059a0/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L177
+SUBDOMAIN_NAME = "^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"
+ALWAYS_ALLOWED = ["$(resource-field)", "$(todo-annotation)"]
```

Comment:
todo: double check we are not allowing the execution of args that include these vars that are always allowed. Before, these placeholders where replaced in the policy and there wasn't a special rule to allow these for args, like there was for node-name.

Reply:
solved by https://github.com/microsoft/kata-containers/pull/316 and rebased